docs/topics_5/proc_web-downloading-logs-console.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+[id="proc_downloading-logs-console_{context}"]
+= Downloading pod logs using the OpenShift Container Platform console
```

Comment:
Verbose. Change to "Downloading logs by using the OpenShift web console"

That is the proper name of the console.